app/assets/javascripts/components/samples.jsx

```diff
@@ -144,71 +177,102 @@ class Samples extends React.Component {
-              </p>
-            </div>
-              <div className={ runInfo.total_runtime ? "reads status-col col s2" : 'reads col s2 no-time'}>{ !runInfo.job_status_description ? rowWithChunkStatus : rowWithoutChunkStatus }
-              { runInfo.total_runtime ? <p className="time"><i className="fa fa-clock-o" aria-hidden="true"></i><span>{this.formatRunTime(runInfo.total_runtime)}</span></p> : ''}
+        <a key={i} href={`/samples/${dbSample.id}`}>
+          <div className='col s12 no-padding sample-feed'>
+            <div className='samples-card white'>
+              <div className='flex-container'>
+                <ul className='flex-items'>
+                  <li className='sample-name-info'>
+                    <div className='card-label top-label'>
+                      {/*<span className='project-name'>*/}
+                      {/*Mosquito*/}
+                      {/*</span>*/}
+                      <span className='upload-date'>
+                        Uploaded {moment(dbSample.created_at).startOf('second').fromNow()}
+                      </span>
+                    </div>
+                    <div className='card-label center-label sample-name'>
+                      {dbSample.name}
+                    </div>
+                    <div className='card-label author bottom-label author'>
+                      { !uploader || uploader === '' ? '' : <span>Uploaded by: {uploader}</span>}
+                    </div>
+                  </li>
+                  {
+                    this.state.columnsShown.map((column, pos) => {
+                      const nonhost_reads_percent = (column === 'nonhost_reads')
+                        ? data_values["nonhost_reads_percent"] : '';
+                      let column_data = '';
+                      if (column === 'pipeline_status') {
+                        column_data = (
+                          <li  key={pos}>
+                            <div className='card-label top-label'>
+                              { rowWithChunkStatus }
+                            </div>
+                            <div className='card-label center-label'>
+                              { runInfo.total_runtime ?
+                                <span className="time">
+                                  <i className="fa fa-clock-o" aria-hidden="true"/>
+                                  <span className='duration-label'>
+                                    { this.formatRunTime(runInfo.total_runtime) }
+                                    </span>
+                                </span> : null
+                              }
+                            </div>
+                          </li>
+                        )
+                      } else {
+                        column_data = (<li key={pos}>
+                          <div className='card-label center center-label'>
+                            {data_values[column]} {nonhost_reads_percent}
```

Comment:
from a readability perspective, probably better to explicitly check three cases here (column === 'pipeline_status', column === 'nonhost_reads', column === anything else) rather than appending a variable named "nonhost_reads_percent" to all columns :)